apps/web/src/views/Swap/hooks/useFeeSaved.ts

```diff
@@ -36,11 +38,19 @@ export const useFeeSaved = (inputAmount?: CurrencyAmount<Currency>, outputAmount
   const { data: outputCurrencyUSDPrice } = useCurrencyUsdPrice(feeSavedAmount?.currency, {
     enabled: Boolean(feeSavedAmount),
   })
+  const fallbackPrice = useStablecoinPrice(feeSavedAmount?.currency, {
+    enabled: Boolean(feeSavedAmount) && !outputCurrencyUSDPrice,
```

Comment:
shall we only enable after trying with api while there's no valid result?

Reply:
sure, add a `isLoading` to enabled check it